apps/etterlatte-pdltjenester/src/test/kotlin/person/PersonServiceTest.kt

```diff
@@ -196,6 +198,9 @@ internal class PersonServiceTest {
                 )
             }
         val expectedFolkeregisterIdent = "70078749472"
+        if (personIdentResponse !is PdlIdentifikator.FolkeregisterIdent) {
```

Comment:
Vi kan koste på oss en test her for npid ? 🤠

Reply:
aight, jeg skal teste med en Npid også :D